### DISCUSSION TITLE
Redirect on login for a group doesn't work if multiple languages are enabled

### DISCUSSION THREAD

**Discussion by hryhorash** [requarks/wiki#4470] [Error / Bug Report]
I've set up redirects to the appropriate pages for different user groups. Everything worked fine until I activated multi language support.
Now on login all users are redirected to the main page.
I tried to specify page for redirect in group settings with locale and without - doesn't work in any case (

Am I missing something? Looks like a bug

Wiki.js 2.5.201

**Comment by Zuline** [comment]
This is exactly the case with 2.5.289

The only way I have been able to get redirect to work is by switching OFF multi-lingual namespaces. I didn't change any permissions, it just worked as soon as I turned off multi-lingual namespaces.

Very frustrating

**Comment by KlingoTitz** [comment]
I have the same issue with version 2.5.298. Searched now two days for this solution. After switching off multilanguage the forwarding for the group is working. Strange behaviour.

**Comment by cp-deha** [comment]
Issue still exists in 2.5.300.

**Comment by steve-orbussoftware** [comment]
Issue for us too, and we can't revert the language settings now due to all the cross links containing the language path within the wiki.

**Comment by kiliw** [comment]
Issue is still present in version 2.5.305. 

Tried the group login redirect first with multilangue support but it did not work. Also checked the tipp of @NGPixel in https://www.reddit.com/r/wikijs/comments/17c6aln/group_redirect_on_login_not_working/

Yet, after switching off the multilanguage, the redirect worked as expected. Please fix this, since we need multiple languages.
